collection-input/cesm1-le-collection.yml

```diff
@@ -21,6 +21,15 @@ data_sources:
             member_id: 0
             has_ocean_bgc: True
             year_offset: 1448
+          - case: e.e11.E1850C5CN.f09_g16.001
```

Comment:
These (e.e11.E1850C5CN.f09_g16.001 and f.e11.F1850C5CN.f09_f09.001) do *not* belong as additional ensemble members of the `CTRL`!

F case is atmosphere only; an E case is with the slab ocean model.

http://www.cesm.ucar.edu/experiments/cesm1.1/LE/

f.e11.F1850C5CN.f09_f09.001 could be called CTRL_AMIP
e.e11.E1850C5CN.f09_g16.001 could be called CTRL_SLAB_OCN

Reply:
You will note that there are additional categories of experiment linked on the page referenced above. These could be included too—but I suggest we wait for a motivated user.

Reply:
Sounds good! I just need these two:

> f.e11.F1850C5CN.f09_f09.001 could be called CTRL_AMIP
e.e11.E1850C5CN.f09_g16.001 could be called CTRL_SLAB_OCN

for the AWS transfer.